main.go

```diff
@@ -4,27 +4,45 @@ import (
 	"github.com/gin-gonic/gin"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/mozilla/doorman/config"
 	"github.com/mozilla/doorman/doorman"
 	"github.com/mozilla/doorman/utilities"
 )
 
+func init() {
+	config.AddLoader(&config.FileLoader{})
+	config.AddLoader(&config.GithubLoader{
+		Token: settings.GithubToken,
+	})
+}
+
 func setupRouter() (*gin.Engine, error) {
```

Comment:
`setupRouter()` doesn't seem necessary as it's only used in `main()`.  It's probably fine to just put all this logic right into `main()`.  This isn't a big deal.

Reply:
oh i see you use `setupRouter()` so you can test that it works. NM.